openmc/mgxs/mgxs.py

```diff
@@ -10,6 +10,11 @@
 
 import numpy as np
 
+# Require numpy to print output in scientific notation to 6 decimal places.
+# This is needed to avoid round off error when large numbers are printed,
+# which can cause tests to fail for different build configurations.
+np.set_printoptions(formatter={'float': lambda x: format(x, '8.6E')})
```

Comment:
I'm curious whether this option can be set in our test suite *rather* than setting it universally for the API? Perhaps it can be set in the ["testing_harness.py"](https://github.com/mit-crpg/openmc/blob/develop/tests/testing_harness.py) file?